prusti-tests/tests/compiletest.rs

```diff
@@ -102,7 +102,6 @@ fn run_prusti_tests(group_name: &str, filter: &Option<String>, rustc_flags: Opti
 fn run_no_verification(group_name: &str, filter: &Option<String>) {
     let _temporary_env_vars = (
         TemporaryEnvVar::set("PRUSTI_FULL_COMPILATION", "true"),
-        TemporaryEnvVar::set("PRUSTI_NO_VERIFY", "true"),
```

Comment:
So what makes Prusti skip verification for these tests?

Reply:
I had to fix the prusti-driver adding `flags.skip_verify = config::no_verify();`, because this configuration flag was not used at all. I forgot to revert this deletion, CI catched it.